schemas/SvrPropertiesInDeviceState.json

```diff
@@ -0,0 +1,103 @@
+{
+"key":{
+    "R": "Read Only",
+    "RW": "Read Write",
+    "-": "No Access",
+    "": "Unspecified"
+},
+"doxm":{
+    "DOXS":{
+        "oxms": {"RESET": "R", "RFOTM": "R", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "oxmsel": {"RESET": "R", "RFOTM": "RW", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "sct": {"RESET": "R", "RFOTM": "R", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "owned": {"RESET": "R", "RFOTM": "RW", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "deviceuuid": {"RESET": "R", "RFOTM": "RW", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "devowneruuid": {"RESET": "R", "RFOTM": "RW", "RFPRO":"R", "RFNOP":"R", "SRESET": "R"},
+        "rowneruuid": {"RESET": "R", "RFOTM": "RW", "RFPRO":"R", "RFNOP":"R", "SRESET": "RW"}},
+    "CMS":{
+        "oxms": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "oxmsel": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "sct": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "owned": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "deviceuuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "devowneruuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "rowneruuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""}},
+    "AMS":{
+        "oxms": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "oxmsel": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "sct": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "owned": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "deviceuuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "devowneruuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""},
+        "rowneruuid": {"RESET": "", "RFOTM": "", "RFPRO":"", "RFNOP":"", "SRESET": ""}},
+    "CLIENT_A":{
```

Comment:
Who is "Client_A"?  Shouldn't this be "Anonymous Client" and "Authenticated Client" (two separate entries)?